### PR TITLE
Core: Remove webpack DLLs

### PR DESCRIPTION
**Pull request by shilman**
Issue: #12637

## What I did

Remove prebuilt DLLs

## How to test

CI passes

### CODE REVIEW

**Review by ScriptedAlchemy — APPROVED** [review]

**Comment by ScriptedAlchemy** [comment]
We will be replacing that with something a little better soon ;)

File: docs/api/cli-options.md [hunk 1]
```diff
@@ -43,7 +41,7 @@ Usage: build-storybook [options]
 
     <summary><h4>Troubleshooting routing issues with Storybook 6.0</h4></summary>
 
-    If you are building your Storybook and you encounter an issue where you cannot change the route in the sidebar, try building Storybook with the `--no-dll` flag and see if it solves the problem. If so, adjust your `build-storybook` script accordingly to include this flag. We would like to point out that your build process will run slower than usual when using this flag. 
+    If you are building your Storybook and you encounter an issue where you cannot change the route in the sidebar, try building Storybook with the `--no-dll` flag and see if it solves the problem. If so, adjust your `build-storybook` script accordingly to include this flag. We would like to point out that your build process will run slower than usual when using this flag.
```
Comment (tmeasday) [inline docs/api/cli-options.md:44]: Doesn't this do nothing now?

**Review by tmeasday — APPROVED** [review]
OK, LGTM generally

**Comment by shilman** [comment]
self-merging @tmeasday @ndelangen @ghengeveld

**Pull request merged**